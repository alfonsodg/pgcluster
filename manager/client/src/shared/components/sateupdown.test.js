import React from 'react';
import { shallow } from 'enzyme';

import StateUpDown from './stateupdown.js';

describe('StateUpDown', () => {
  it('should render correctly', () => {
    const expectedStyle = {
      "color": "orange",
      "fontWeight": "bold",
    }
    const component = shallow(<StateUpDown color="orange" />);
  
    expect(component).toMatchSnapshot();
    /* implementation detail ?
    expect(component.text()).toEqual('X');
    expect(component.prop('style')).toEqual(expectedStyle);
    */
  });
});