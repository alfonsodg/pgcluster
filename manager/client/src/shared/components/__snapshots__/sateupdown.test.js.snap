// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StateUpDown should render correctly 1`] = `
<div
  style={
    Object {
      "color": "orange",
      "fontWeight": "bold",
    }
  }
>
  X
</div>
`;
